src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerContactParties.java

```diff
@@ -94,7 +95,7 @@ private void doConfigure(final ConfigBuilder<CaseData, State, UserRole> configBu
                 .submittedCallback(this::partiesContacted)
                 .grant(CREATE_READ_UPDATE, SUPER_USER,
                     ST_CIC_CASEWORKER, ST_CIC_SENIOR_CASEWORKER, ST_CIC_HEARING_CENTRE_ADMIN,
-                    ST_CIC_HEARING_CENTRE_TEAM_LEADER, ST_CIC_SENIOR_JUDGE)
+                    ST_CIC_HEARING_CENTRE_TEAM_LEADER, ST_CIC_SENIOR_JUDGE, ST_CIC_RESPONDENT)
                 .grantHistoryOnly(
```

Comment:
There is another event Called RespondentContactParties. Would not be an issue?

Reply:
ok Elif, haven't realized that there is a separate class for Respondent. So this is change is not needed anymore.